BlogTemplate/Pages/Edit.cshtml.cs

```diff
@@ -60,6 +63,7 @@ public IActionResult OnPostSaveDraft()
             return Redirect("/Index");
         }
 
+        [ValidateAntiForgeryToken]
         public void UpdatePost(Post newPost, string slug)

```

Comment:
This should not be a browser-invokable method.  Remove the attribute and make it private instead.